pkg/customers/customers.go

```diff
@@ -104,18 +105,20 @@ func respondWithCustomer(logger log.Logger, w http.ResponseWriter, customerID st
 // TODO(adam): What GDPR implications does this information have? IIRC if any EU citizen uses
 // this software we have to fully comply.
 type customerRequest struct {
-	FirstName  string              `json:"firstName"`
-	MiddleName string              `json:"middleName"`
-	LastName   string              `json:"lastName"`
-	NickName   string              `json:"nickName"`
-	Suffix     string              `json:"suffix"`
-	Type       client.CustomerType `json:"type"`
-	BirthDate  model.YYYYMMDD      `json:"birthDate"`
-	Email      string              `json:"email"`
-	SSN        string              `json:"SSN"`
-	Phones     []phone             `json:"phones"`
-	Addresses  []address           `json:"addresses"`
-	Metadata   map[string]string   `json:"metadata"`
+	CustomerID string                `json:"-"`
+	FirstName  string                `json:"firstName"`
+	MiddleName string                `json:"middleName"`
+	LastName   string                `json:"lastName"`
+	NickName   string                `json:"nickName"`
+	Suffix     string                `json:"suffix"`
+	Type       client.CustomerType   `json:"type"`
+	BirthDate  model.YYYYMMDD        `json:"birthDate"`
+	Status     client.CustomerStatus `json:"status"`
```

Comment:
Can we have this be `json:"-"`? We have a separate endpoint for updating a Customer's status.

Reply:
Updated!